.eslintrc.json

```diff
@@ -62,10 +62,14 @@
       ],
       "plugins": [
         "jest"
+      ]
+    },
+    {
+      "files": [
+        "**/*.spec.{ts,tsx}"
```

Comment:
:bulb: `setup-jest.js` uses test hooks outside of describe blocks, so we make the `jest/require-top-level-describe` rule target only spec files.